src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.username=freeopenuniversity
-#spring.datasource.password=mysqlpass
+spring.datasource.username=root
+spring.datasource.password=root
+server.port=8082
```

Comment:
Is there a reason to change username, password and port number.

Reply:
The latest one should be the follwoing,  which is what is in the latest develop branch.   I used Env variable if supplied,  if not use what you had before.  "OpenFreeUniversity:"

spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQLDialect
spring.datasource.username=${DATABASE_USER:OpenFreeUniversity}
spring.datasource.password=${DATABASE_PASSWORD:}
spring.datasource.url= ${DATABASE_URL:jdbc:mysql://localhost:3306/test}